### DISCUSSION TITLE
ppocrv4 det 微调

### DISCUSSION THREAD

**Discussion by rongyixiao180319** [PaddlePaddle/PaddleOCR#12746] [Q&A]
自己的数据：训练数据接近四千张，验证集100张，遇到两个问题：
1、ppocrv4 det 微调为什么没有原始模型效果好，自己训练的模型精度precision:0.76，recall:0.92；
2、同样的数据，ppocrv2 det微调反而比ppocrv4 det效果好，这是为什么？用ppocrv2 det 自己训练的模型精度precision:0.98，recall:0.99。
求解答，谢谢！

**Comment by GreatV** [comment]
- 数据量和多样性：训练数据的数量和多样性可能不足，导致模型没有充分学习到特征。

- 数据质量：数据的标注质量和图像的清晰度也会影响训练效果。如果数据中存在噪声或标注错误，模型的性能会受到影响。

- 超参数调整：模型微调时的超参数设置（如学习率、batch size等）可能不适合你的数据集。可以尝试不同的超参数设置以获得更好的效果。
